x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java

```diff
@@ -83,8 +83,8 @@ public void testFullPolicy() throws Exception {
         // index document {"foo": "bar"} to trigger rollover
         index(client(), originalIndex, "_id", "foo", "bar");
         assertBusy(() -> assertTrue(indexExists(secondIndex)));
-        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
         assertBusy(() -> assertFalse(indexExists(originalIndex)));
+        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
```

Comment:
If the order of these asserts is important in a non-obvious way, we should probably include a comment explaining why.